### ISSUE TITLE
"special" tags are being removed on reedit

### ISSUE THREAD

**Issue by agitator** [plone/plone.app.mosaic#160]
i configure the richtext tile so that html-snippets can be inserted
i also configured the html-cleanup and tinymce to allow embed/script tags

when i insert the snippets it is displayed correctly after save.
on reedit the whole area is empty

if there is some other text in a paragraph, the snippets stays as it is
a fixed blank within a p tag on the other hand is not enough

**Comment by datakurre** [comment]
I agree that this is probably a bug. Mosaic should use Plone TinyMCE configuration nowadays.

@agitator Could you add a minimal example of the configuration and inserted data into this issue.

**Comment by agitator** [comment]
config
```
{
"extended_valid_elements":"script[language|src|width|height]"
}
```

snippet, 
```
<script height="355px" width="500px" src="http://player.some.url"></script>
```

**Comment by thet** [comment]
probably unrelated, but a script tag cannot have width and height arguments.

**Comment by agitator** [comment]
at least it doesn't matter for that problem
